monitoring/dashboards/README.md

```diff
@@ -0,0 +1,30 @@
+# Building new dashboards
+
+## From OSS
+
+1. Download the OSS JSON definition
+2. Import it in a deployed Grafana
+3. Tweak to your liking
+
+## From scratch
+
+1. Create a new dashboard in a deployed Grafana
+2. Create and tweak panels
+
+
+## Final steps
+
+1. Copy the JSON model
+2. Apply the following modifications:
+   * add the `__inputs` of the original download to the model
```

Comment:
would it be relevant to document what should (typically) be as `inputs` and what should be as `variables` ?
Something like :

* inputs should be used match the deployment: e.g. datasource, namespace & service name to filter on...
* variables should be used to select between nodes or instances of a services